### PR TITLE
reword the "invalid password" dialogue

### PR DESCRIPTION
**Pull request by hboetes**

### CODE REVIEW

File: pynicotine/gtkgui/application.py [hunk 1]
```diff
@@ -344,8 +344,8 @@ def on_invalid_password(self):
         from pynicotine.gtkgui.widgets.dialogs import OptionDialog
 
         title = _("Invalid Password")
-        msg = _("User %s already exists, and the password you entered is invalid. Please choose another username "
-                "if this is your first time logging in.") % config.sections["server"]["login"]
+        msg = _("Either user %s already exists, or the password you entered is invalid."
+                " If this is the first time you log in, please choose a different username.") % config.sections["server"]["login"]
```
Comment (slook) [inline pynicotine/gtkgui/application.py:348]: "...time you log in" doesn't make sense.

**Review by slook — CHANGES_REQUESTED** [review]

File: pynicotine/gtkgui/application.py (hunk 1 above)
Comment (hboetes) [reply pynicotine/gtkgui/application.py:348]: How would you put it?

Comment (mathiascode) [reply pynicotine/gtkgui/application.py:348]: What was wrong with `your first time logging in`?

File: pynicotine/gtkgui/application.py [hunk 2]
```diff
@@ -344,8 +344,8 @@ def on_invalid_password(self):
         from pynicotine.gtkgui.widgets.dialogs import OptionDialog
 
         title = _("Invalid Password")
-        msg = _("User %s already exists, and the password you entered is invalid. Please choose another username "
-                "if this is your first time logging in.") % config.sections["server"]["login"]
+        msg = _("Either user %s already exists, or the password you entered is invalid."
```
Comment (mathiascode) [inline pynicotine/gtkgui/application.py:347]: A password can only be invalid if the user exists.

File: pynicotine/gtkgui/application.py (hunk 1 above)
Comment (hboetes) [reply pynicotine/gtkgui/application.py:348]: Strictly speaking there isn't, but the proposed construction is how a native speaker would formulate it.

Comment (slook) [reply pynicotine/gtkgui/application.py:348]: An alternative way of putting it is `If this is the first time you have logged in`.

**Comment by slook** [comment]
I don't think we should bother updating the invalid password string at the moment. The entire mechanism requires deeper review.

File: pynicotine/gtkgui/application.py (hunk 2 above)
Comment (slook) [reply pynicotine/gtkgui/application.py]: "right" -> "correct"

**Pull request closed without merging**